pkg/dashboard/templates/cost_settings.gohtml

```diff
@@ -0,0 +1,51 @@
+{{ define "cost_settings" }}
+<aside class="cost-settings-box" id="cost-settings-box">
+  <details open>
+    <summary class="--hideMarker">
+      <div class="cost-settings__title-container">
+        <p class="cost-settings-box__toggle-title">
+          Cost Settings
+          <i aria-hidden="true" class="fas fa-fw fa-angle-down --showWhenClosed"></i
+          ><i aria-hidden="true" class="fas fa-fw fa-angle-up --showWhenOpen"></i>
+        </p>
+        <div>
+         <button id="cost-settings__disable-cost-settings">Disable Cost Settings</button>
+        </div>
+      </div>
+    </summary>
+
+    <form class="cost-settings-box__content">
+      <p>Choose an AWS or GCP instance type to get an accurate cost data, or set your cost per CPU and cost per GB of memory manually. Pick "Other" Cloud Provider to customize the numbers.</p>
```

Comment:
This text doesn't seem to be read by the screenreader in my testing. It could be that I'm using it wrong, but I don't seem to be able to find this description.